Modules/IO/ImageBase/include/itkImageSeriesReader.hxx

```diff
@@ -436,17 +437,19 @@ ImageSeriesReader<TOutputImage>::GenerateData()
               outputSpacing[this->m_NumberOfDimensionsInImage])) // either non-uniform sampling or missing slice
         {
           nonUniformSampling = true;
+          spacingDeviation = Math::abs(outputSpacing[this->m_NumberOfDimensionsInImage] - dirNnorm);
           itkWarningMacro(<< "Non unform sampling or missing slices detected , expected "
-                          << outputSpacing[this->m_NumberOfDimensionsInImage] << " got: " << dirNnorm);
+                          << outputSpacing[this->m_NumberOfDimensionsInImage] << " got: " << dirNnorm
+                          << " Deviation of:" << spacingDeviation);
 
-          EncapsulateMetaData<bool>(output->GetMetaDataDictionary(),
-                                    "ITK_non_uniform_sampling",
-                                    true); // set metadata for the series reader output
+          // EncapsulateMetaData<bool>(output->GetMetaDataDictionary(),
```

Comment:
Either leave this piece of code, or remove it. Commenting it out like this might confuse people in the future.